build.gradle

```diff
@@ -30,6 +31,12 @@ allprojects {
   targetCompatibility = '11'
 }
 
+lombok {
+  config['config.stopBubbling'] = 'true'
+  config['lombok.addLombokGeneratedAnnotation'] = 'true'
+  config['lombok.anyConstructor.addConstructorProperties'] = 'true'
```

Comment:
Do we need this if we are not using the Lombok constructor annotations?

Reply:
good point!

Reply:
done.